lib/rubocop/cop/packaging/gemspec_git.rb

```diff
@@ -38,9 +38,10 @@ module Packaging
       #   good_foo_method(args)
       #
       class GemspecGit < Cop
-        MSG = "Don't use git."\
-              "It's a problem while maintaining it downstream."\
-              "Rather use some pure Ruby alternative, like `Dir` or `Dir.glob`"
+        MSG = "Avoid using git to produce lists of files." \
+          "Downstreams often need to build your package in an environment " \
+          "that does not have git (on purpose)." \
+          "Use some pure Ruby alternative, like `Dir` or `Dir.glob`"
```

Comment:
Hi,
Thanks for this. However, while testing this locally, it renders like this:
![image](https://user-images.githubusercontent.com/31288318/84465661-3951fb00-ac95-11ea-88f8-a6c330474734.png)
Although I can do it myself, but while you're at it, can you add spaces at the end of the sentences and a fullstop at the end?

And also use single-quotes instead? :see_no_evil: 
This throws a bunch of "offenses":
```
lib/rubocop/cop/packaging/gemspec_git.rb:41:15: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
        MSG = "Avoid using git to produce lists of files." \
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/packaging/gemspec_git.rb:42:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
          "Downstreams often need to build your package in an environment " \
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/packaging/gemspec_git.rb:43:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
          "that does not have git (on purpose)." \
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/packaging/gemspec_git.rb:44:11: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
          "Use some pure Ruby alternative, like `Dir` or `Dir.glob`"
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```